desc_source.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/jhump/protoreflect/desc/protoprint"
```

Comment:
can you move this down into the next import block?

Reply:
solved